### ISSUE TITLE
TypeScript: `config` is required in one overload when all config options are optional in v5

### ISSUE THREAD

**Issue by devjmetivier** [muxinc/mux-node-sdk#163]
v4: 
```ts
new Mux(token, secret);
// ☝🏼 no TS errors
```

v5: 
```ts
new Mux(token, secret);
// ☝🏼  TS Error: No overload expects 2 arguments, but overloads do exist that expect either 1 or 3 arguments.

// 👇🏼 This is required
new Mux(token, secret, {});
```

**Comment by eropple** [comment]
Hey, thanks for filing an issue! I've got a PR out, we'll get this sorted.

**Comment by eropple** [comment]
#164 is released as **5.0.1**.

Thanks!
